frontend/src/components/core/Block/Block.tsx

```diff
@@ -256,12 +255,10 @@ class Block extends PureComponent<Props> {
     return (
       <Maybe enable={enable} key={key}>
         <StyledElementContainer
-          data-stale={isStale}
+          data-stale={!!isStale}
```

Comment:
Why does it need to be cast to bool?

Reply:
In the test we assume the value is either `"true"` or `"false"` (e.g. `.should("have.attr", "data-stale", "false")` so I didn't want it to print values like `null` or `undefined`. I didn't investigate where `isStale` comes from though, not sure if it's already guaranteed to be a bool.